src/crepe/Sound.cpp

```diff
@@ -0,0 +1,60 @@
+#include "util/log.h"
+
+#include "Sound.h"
+#include "SoundContext.h"
+
+using namespace crepe;
+
+Sound::Sound(std::unique_ptr<api::Resource> res) {
+	dbg_trace();
+	this->load(std::move(res));
+}
+
+Sound::Sound(const char * src) {
+	dbg_trace();
+	this->load(std::make_unique<api::Resource>(src));
```

Comment:
deze moeten uiteindelijk nog verandert worden met niels nieuwe resource idee toch?

Reply:
Op deze branch heet de klasse nog `crepe::api::Resource` maar ik merge zo `loek/cleanup` waar deze hernoemd is naar `crepe::Asset`. De *nieuwe* resource manager beheert straks de instantie van `crepe::Sound`, en hier is Sound zelf niet bewust van.